### ISSUE TITLE
Proposed Feature: Static Category Link Buttons (Remix)

### ISSUE THREAD

**Issue by UnKnoWn-Consortium** [nyaadevs/nyaa#128]
There have been requests for the return of the static category link buttons in the old Nyaa (as in #106 and even in #92 ). I am proposing the following redesign that would be both modern and colorful :)

> ![2017-05-19_024810](https://cloud.githubusercontent.com/assets/9733856/26218461/a1f83c30-3c3d-11e7-8519-9b9a85f00946.png)
> ![2017-05-19_022347](https://cloud.githubusercontent.com/assets/9733856/26217591/8a16e7e0-3c3a-11e7-86b5-9ff5977a57cc.png)
> ![2017-05-19_022705](https://cloud.githubusercontent.com/assets/9733856/26217647/b454c324-3c3a-11e7-8bde-84f8b81db81a.png)

Such design can also include a more complicated control flow. A small open button, which will reveal the subcategories upon clicks, can be added to the end each parent category link button. Though it may slightly duplicate the function provided by the search bar. 

P.S. The background colors are for reference only.

**Comment by JLChamberlain** [comment]
I prefer this over having just words like the original site but I think the size of them kind of overwhelms the rest of the page. Maybe have them smaller in height, keep the length and make them only appear when hovered over to make the design look cleaner and sharper. Thoughts?

**Comment by nx6** [comment]
Call me crazy, but doesn't this browsing feature already exist as part of the search function?
Leave the actual search field blank, and set your filter/category options. Now search.
The result is the site with the other stuff removed. Now bookmark the URL you're on.

Example: Eng-trans Anime with no remakes: https://nyaa.si/?q=&f=1&c=1_2

**Comment by JLChamberlain** [comment]
Yes you can do that way but the original site had the category shortcut buttons at the top as a quicker way of doing it. This is why people are suggesting it instead of having to type nothing in the search bar.

**Comment by UnKnoWn-Consortium** [comment]
@nx6 I am well aware of that function. That's why I have mentioned this design might be a little bit duplicating what the search bar is capable of. However, I am still proposing this because (1) the old Nyaa had it and it appears to be pretty popular among users, (2) embedding such function inside the search bar is quite counter-intuitive UX-wise., and (3) these static category buttons are shortcuts that save a few clicks.
